src/App.tsx

```diff
@@ -10,33 +11,19 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(0);
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is Tab ${selectedTabId + 1}`}
```

Comment:
so, when you have a tab at state, you can use 
```suggestion
        {`Selected tab is Tab ${selectedTab.title}`}
```